modules/docs/arrow-docs/docs/docs/arrow/typeclasses/repeat/README.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+title: Repeat
+permalink: /docs/arrow/typeclasses/repeat/
+redirect_from:
+  - /docs/typeclasses/repeat/
+---
+
+## Zip
+
+{:.beginner}
+beginner
+
+The `Repeat` typeclass extends the `Zip` typeclass with an repeat structure.
```

Comment:
```suggestion
The `Repeat` typeclass extends the `Zip` typeclass with a way to repeat the structure.
```
The previous sounded like `Repeat` adds any capability to the datatype itself when it actually just provides methods for that datatype

Reply:
+